clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -44,12 +43,14 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
 
   private def verifyCloudPaths(ioUtil: IoUtil): Unit = {
     if (moveCommand.key.location != Location.GCP) {
-      throw new Exception(s"Only cloud ${name}s are supported at this time.")
+      throw new IllegalArgumentException(
+        s"Only cloud ${name}s are supported at this time."
+      )
     }
 
-    if (!ioUtil.isGoogleObject(destination)) {
-      throw new Exception(
-        s"The destination of the $name must be a cloud path. '$destination' is not a cloud path."
+    if (!ioUtil.isGoogleDirectory(destination)) {
```

Comment:
This also changed: it used to accept non-directory destinations if there was only one path associated with the document to move, but now the move destination always has to be a directory. I made the change to remove a bunch of special cases in the logic, and thought it'd be OK because:

1. Only wgs-ubams track a single file, so most moves require passing a directory anyways
2. We're not moving wgs-ubams anywhere yet, so nothing will be broken by this